client/app/readerprototype/components/ReaderFooter.jsx

```diff
@@ -17,10 +30,17 @@ const ReaderFooter = ({
   showNextDocument,
   showPreviousDocument,
 }) => {
+
   const handleKeyPress = (event) => {
     if (event.key === 'Enter') {
-      document.getElementById(`canvas-${event.target.value}`).scrollIntoView();
-      setCurrentPage(event.target.value);
+      const targetPage = event.target.value;
+
+      if (validatePageNum(targetPage, numPages)) {
+        document.getElementById(`canvas-${targetPage}`).scrollIntoView();
+        setCurrentPage(targetPage);
+      } else {
```

Comment:
i suppose just for paranoia's sake you could do
```suggestion
      } else if (currentPage) {
```
just in case currentPage is null or 0 or undefined